spec/runtime/setup_spec.rb

```diff
@@ -108,6 +108,38 @@
     end
   end
 
+  context "load order" do
+    after do
```

Comment:
Addressed in https://github.com/Elffers/bundler/commit/10ccab6a0dae411d722925fb9030475fd315db01